pkg/cloud/ibm/assets/deployment.yaml

```diff
@@ -0,0 +1,114 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ibm-cloud-controller-manager
+  namespace: openshift-cloud-controller-manager
+  labels:
+    k8s-app: ibm-cloud-controller-manager
+spec:
+  replicas: 3
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+      maxSurge: 3
+  selector:
+    matchLabels:
+      k8s-app: ibm-cloud-controller-manager
+  template:
+    metadata:
+      labels:
+        k8s-app: ibm-cloud-controller-manager
+    spec:
+      priorityClassName: system-cluster-critical
+      hostNetwork: true
+      nodeSelector:
+        node-role.kubernetes.io/master: ""
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - topologyKey: "kubernetes.io/hostname"
+            labelSelector:
+              matchLabels:
+                k8s-app: ibm-cloud-controller-manager
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+        operator: Exists
+      - effect: NoExecute
+        key: node.kubernetes.io/unreachable
+        operator: Exists
+        tolerationSeconds: 120
+      - effect: NoExecute
+        key: node.kubernetes.io/not-ready
+        operator: Exists
+        tolerationSeconds: 120
+      - effect: NoSchedule
+        key: node.cloudprovider.kubernetes.io/uninitialized
+        operator: Exists
+      - effect: NoSchedule
+        key: node.kubernetes.io/not-ready
+        operator: Exists
+      serviceAccountName: cloud-controller-manager
```

Comment:
The default RBAC for this service account is not sufficient for the IBM CCM.  We'll either need a base RBAC update or a custom addition for the IBM CCM.  I assume that the later will be preferred.

Reply:
Looks like we can update similar to https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/81

Reply:
https://github.com/openshift/cluster-cloud-controller-manager-operator/pull/100 should fix